src/Resources/contao/dca/tl_style_manager.php

```diff
@@ -222,3 +222,14 @@
         ]
     ]
 ];
+
```

Comment:
@c4y Could you rework this as a callback within an event listener?

Reply:
sure